requirements.txt

```diff
@@ -10,4 +10,3 @@ negotiator==1.0.0
 psycopg2==2.6
 requests==2.4.0
 static3==0.5.1
-wheel==0.24.0
```

Comment:
Note, I don't think it's a good idea to lock wheel here because wheel is a package for installing packages and it is backwards being installed here as it will be installed /with/ the packages and the version may differ.